datasets/BBC_Hindi_NLI/BBC_Hindi_NLI.py

```diff
@@ -0,0 +1,151 @@
+"""
+
+# TODO: Add a link to an official homepage for the dataset here
+_HOMEPAGE = "https://github.com/midas-research/hindi-nli-data"
+
+# TODO: Add the licence for the dataset here if you can find it
+_LICENSE = """
+MIT License
+
+Copyright (c) 2019 MIDAS, IIIT Delhi
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+"""
+
+_TRAIN_DOWNLOAD_URL = "https://raw.githubusercontent.com/midas-research/hindi-nli-data/master/Textual_Entailment/BBC/BBC_recasted_train.tsv"
+_VALID_DOWNLOAD_URL = "https://raw.githubusercontent.com/midas-research/hindi-nli-data/master/Textual_Entailment/BBC/BBC_recasted_dev.tsv"
+_TEST_DOWNLOAD_URL = "https://raw.githubusercontent.com/midas-research/hindi-nli-data/master/Textual_Entailment/BBC/BBC_recasted_test.tsv"
+
+
+class BBC_Hindi_NLI_Config(datasets.BuilderConfig):
+    """BuilderConfig for BBC Hindi NLI Config"""
+
+    def __init__(self, **kwargs):
+        """BuilderConfig for BBC Hindi NLI Config.
+        Args:
+          **kwargs: keyword arguments forwarded to super.
+        """
+        super(BBC_Hindi_NLI_Config, self).__init__(**kwargs)
+
+
+class BBC_Hindi_NLI(datasets.GeneratorBasedBuilder):
+    """BBC Hindi NLI dataset -- Dataset providing textual-entailment pairs for NLI tasks in Hindi"""
+
+    BUILDER_CONFIGS = [
+        BBC_Hindi_NLI_Config(
+            name="BBC Hindi NLI",
+            version=datasets.Version("1.1.0"),
+            description="BBC Hindi NLI: Natural Language Inference Dataset in Hindi",
+        ),
+    ]
+
+    def _info(self):
+
+        return datasets.DatasetInfo(
+            description=_DESCRIPTION,
+            features=datasets.Features(
+                {
+                    "Context": datasets.Value("string"),
+                    "Hypothesis": datasets.Value("string"),
+                    "Entailment_Label": datasets.Value("string"),
+                    "Topic_Label": datasets.Value("string"),
```

Comment:
Could you rename the fields `premise`, `hypothesis`, `label`, and `topic` ? This is the standard we use for NLI datasets.
Moreover can you use the ClassLabel feature type for label and topic ?
For example
```python
"label": datasets.ClassLabel(names=["neutral", "entailment", "contradiction"])
"topic": datasets.ClassLabel(names=["india", "pakistan", "news", ...])
```

your'll need to regenerate the dataset_infos.json file after these changes